runtime/asgard/src/lib.rs

```diff
@@ -281,6 +281,8 @@ parameter_types! {
 	pub const BifrostCrowdloanId: PalletId = PalletId(*b"bf/salp#");
 	pub const LiquidityMiningPalletId: PalletId = PalletId(*b"bf/lm###");
 	pub const LighteningRedeemPalletId: PalletId = PalletId(*b"bf/ltnrd");
+	pub const MerkleDirtributorPalletId: PalletId = PalletId(*b"/mkl-dis");
+	pub const StringLimit: u32 = 50;
```

Comment:
The const definition of StringLimit should be moved to the runtime impl of merkle_distributor, since the block here is only for holding different pallet ids.

Reply:
It has been updated